service/requirements.txt

```diff
@@ -0,0 +1,4 @@
+cffi==1.13.2
```

Comment:
This file needs to be accounted for in the MANIFEST_EXEMPT and the two Makefile.am EXTRA_DIST.